_tools/generate_repo.py

```diff
@@ -57,12 +57,9 @@
 # Ignored files and folders:
 ignored_dirs = ['.git', '.idea', '__MACOSX', '.svn', '.vscode']
 ignored_files = ['.gitignore', 'gitattributes', '.gitkeep', '.github']
+ignored_exts = ['.pyc', '.pyo']
 
-
-if ignored_dirs:
-    ignored_dirs.extend([output_path, tools_path])
-else:
-    ignored_dirs = [output_path, tools_path]
+ignored_dirs.extend([output_path, tools_path])
```

Comment:
If ignored_dirs is None, line 62 might throw exception hence I had the conditional.

Reply:
it cant be none since it is defined as list.
https://github.com/Twilight0/Repository-Bootstrapper/blob/master/_tools/generate_repo.py#L58

Reply:
I agree, but some user may change it to None, I don't think that conditional is redundant. I knew its purpose when I added it.

Reply:
What happens if they change it to True, or some non empty string? Then the script would try to iterate over True and raise iterable exception. Or in non empty string situation, the code would iterate each character.

 It is not possible to cover each possibilty unless a predefined type to a config instance us agreed upon which in thus case it is a list. Not a boolean or string or so.e other type.

Reply:
Can't we just agree of the necessity of the conditional regardless if it covers every possible data type a user can enter? It was added there for a simple reason.

Reply:
I actually now came to realize that these values should be in config.ini file not in py file, then it would be must portable

Reply:
I actually thought of this, but moving such settings into the config.ini would introduce these advanced settings to more novice users of the script.